plugin.php

```diff
@@ -135,11 +139,51 @@ protected function __construct() {
         add_action( 'wp_enqueue_scripts', function() {
             wp_enqueue_style( 'dashicons' );
         } );
-	    add_action( 'before_woocommerce_init', function() {
```

Comment:
Need to make sure this branch is up to date with my changes for HPOS as this was added for HPOS but is showing as being removed here